src/components/dashboard/Dashboard.js

```diff
@@ -140,6 +140,7 @@ const Dashboard = props => {
       return
     }
     if (reset) {
+      await delay(2500)
```

Comment:
what is this? reset is called when a new item arrives.

Reply:
@sirpy I added this delay to show the full animation loading as the feeds loads too fast and the animation is replaced with feed

Reply:
@yaroslav-fedyshyn-nordwhale  so everytime there's a new item the animation will show? or there will simply be a delay before feed re-render when new item arrive? meaning you need to check it for example with a payment link.

@LiavGut do we need to wait until the animation ends or can we show the feed once it is ready?